packages/lesswrong/components/users/UsersMenu.tsx

```diff
@@ -263,6 +262,7 @@ const UsersMenu = ({classes}: {
                 iconClassName={classes.icon}
               />
             }
+            {isEAForum && messagesNode}
```

Comment:
Slightly reordered the user menu options on EAF, so that "Saved & read" is right under "User profile" (I think that those two conceptually go together)